test/test_model.py

```diff
@@ -1484,87 +1541,90 @@ def test_unit_get_container(self):
             unit.get_container('c3')
 
         with pytest.raises(RuntimeError):
-            other_unit = self.model.get_unit('other')
+            other_unit = model.get_unit('other')
             other_unit.get_container('foo')
 
 
-class TestContainerPebble(unittest.TestCase):
-    def setUp(self):
+class TestContainerPebble:
+    @pytest.fixture
+    def model(self):
```

Comment:
Do we need `model` as a separate fixture in this class, or can we just merge it into the `container` fixture?